games-arcade/openclaw/openclaw-1.0.3.recipe

```diff
@@ -0,0 +1,61 @@
+SUMMARY="A 2D side-scroller platform video game"
+DESCRIPTION="
+Captain Claw is a 2D side-scroller platform video game \
+published by Monolith Productions in 1997 about the anthropomorphic \
+pirate cat Captain Nathaniel Joseph Claw, who sets on a quest to find \
+an ancient amulet while fighting enemies and solving puzzles."
+HOMEPAGE="https://www.lith.com/games/captain-claw"
+COPYRIGHT="2012 Michael L. McShaffry and David Graham"
+LICENSE="GNU GPL v3"
+REVISION="1"
+srcGitRev="150f7edb7542f3f877b42afc948cc24ea9a46c48"
+SOURCE_URI="https://github.com/pjasicek/OpenClaw/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="9521811f572ca6c61288628cc32ff86f9c7a455afa09387232282f936a531845"
+SOURCE_DIR="OpenClaw-$srcGitRev"
+
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	openclaw$secondaryArchSuffix = $portVersion
+	app:openclaw$secondaryArchSuffix = $portVersion
```

Comment:
As it doesn't build for x86_gcc2, maybe the $secondaryArchSuffix isn't needed here, @fbrosson could give you more info on how to do this the right way) :)

Reply:
@Begasus, I _would have_ suggested using the output of the following command as a template, but I was "told" it was not simple.
```
sed -n -e "s/[Zz]eal/openclaw/; /^ARCHITECTURES=/,/^REQUIRES=/ p" \
	/boot/home/haikuports/app-doc/zeal/zeal-0.6.1.recipe
```
(Feel free to follow what the owners of this repo think is best.)

Reply:
@fbrosson, Are you suggesting to include this command:
```
sed -n -e "s/[Zz]eal/openclaw/; /^ARCHITECTURES=/,/^REQUIRES=/ p" \
	/boot/home/haikuports/app-doc/zeal/zeal-0.6.1.recipe
```
inside PROVIDES? I don't seem to follow.

Reply:
@Vrondir: Sorry, I should have left a less confusing comment.
Here's what you need for your `PROVIDES`:
```
PROVIDES="
	openclaw$secondaryArchSuffix = $portVersion
	app:openclaw = $portVersion
	"
```

Reply:
@fbrosson: Thanks!